doc/releases/changelog-0.28.0.md

```diff
@@ -611,16 +436,40 @@
 * Updated `qml.transforms.metric_tensor`, `qml.transforms.adjoint_metric_tensor`, `qml.qinfo.classical_fisher`, and `qml.qinfo.quantum_fisher` to support the new return types.
   [(#3449)](https://github.com/PennyLaneAI/pennylane/pull/3449)
 
-<h3>Improvements</h3>
+* Updated `qml.transforms.batch_params` and `qml.transforms.batch_input` to support the new return types.
+  [(#3431)](https://github.com/PennyLaneAI/pennylane/pull/3431)
+
+* Updated `qml.transforms.cut_circuit` and `qml.transforms.cut_circuit_mc` to support the new return types.
+  [(#3346)](https://github.com/PennyLaneAI/pennylane/pull/3346)
 
-* The `qml.is_pauli_word` now supports instances of `qml.Hamiltonian`.
+<h3>Improvements 🛠</h3>
+  
+* Multiple mid-circuit measurements can now be combined arithmetically to create new conditionals.
```

Comment:
I think this is worth of being a new feature. E.g., in "New transforms, functions, and more"